### PR TITLE
Set up a hook for useCountry

### PR DESCRIPTION
**Pull request by mbifulco**
The work in this PR will help with the eventual setup of a country switcher.  I created a shorthand for the `<CountryContext.Consumer>` component with a react hook, which is basically an easier-to-read shorthand.  Functionally, this is identical to what was there before - but will help with code readability.

### CODE REVIEW

**Comment by philsturgeon** [comment]
Make sure whatever we do does not conflict with #83. @stephensamra are you ok with Mike running with this one?

File: src/pages/select-your-country.jsx [hunk 1]
```diff
@@ -17,91 +18,84 @@ export default ({ data }) => {
...
-                  <div>
-                    {flag(country.code)} {country.name}
+          ) : null}
+          {Object.keys(countries)
+            .filter(countryCode =>
+              countries[countryCode].toLowerCase().includes(searchTerm)
+            )
+            .map(countryCode => {
+              const countryHasContent = countriesWithContent.includes(
+                countryCode
```
Comment (stephensamra) [inline src/pages/select-your-country.jsx]: I think this might need to be `countryCode.toLowerCase()`

**Comment by stephensamra** [comment]
Thanks @mbifulco, this is definitely a cleaner approach. Will come in handy for #83 👍

File: src/pages/select-your-country.jsx (hunk 1 above)
Comment (mbifulco) [reply src/pages/select-your-country.jsx]: Good catch! Just pushed a fix

**Pull request merged**